packages/playground/.gitignore

```diff
@@ -1 +1,2 @@
 public/libs/
+.vscode/launch.json
```

Comment:
can you remove this line and this file, did you open the playground folder directly in vscode and try to debug?